conftest.py

```diff
@@ -120,6 +138,17 @@ def test_api_obj(request, interactivemode_flag, api_url=base_url_conf.api_base_u
         print("Exception when trying to run test:%s" % __file__)
         print("Python says:%s" % str(e))
 
+def upload_test_logs_to_browserstack(log_name, session_url, appium_test = False):
```

Comment:
Can you add exception handling to ensure any errors are handled. Also can put a check for log file exists too.